src/main/java/doit/apihw/api/service/MemberService.java

```diff
@@ -31,26 +32,48 @@ public MemberResponse findOneMember(Long memberId) {
     // TODO : 전체 회원 정보를 조회한다.
     public List<MemberResponse> findAllMembers() {
         // DB에서 전체 회원 정보를 조회한다.
-
+        List<Member> members = memberRepository.findAll();
         // 조회된 회원 정보를 List<MemberResponse>로 변환하여 반환한다.
-        return null;
+        if(members.isEmpty())
+            throw new IllegalArgumentException("회원이 없음");
+
+        return members.stream()
+                .map(MemberResponse::from)
+                .toList();
     }
 
     // TODO : 회원 이름으로 회원 정보를 조회한다.
     public List<MemberResponse> searchMembersWithName(String memberName) {
         // DB에서 memberName에 해당하는 회원 정보를 조회한다.
+        List<Member> members = memberRepository.findByMemberName(memberName);
+        if(members==null)
+            throw new IllegalArgumentException("해당 회원이 없어여");
 
         // 조회된 회원 정보를 List<MemberResponse>로 변환하여 반환한다.
-        return null;
+        return members.stream()
+                .map(MemberResponse::from)
+                .toList();
     }
```

Comment:
요 부분 꼼꼼하게 체크하는 것도 좋습니다!

여기서 한 가지 생각해보아야할 것이 있습니다.
지금은 회원 이름으로 회원 정보를 조회 요청하였는데 이에 대한 결과가 없는 상황입니다.
1. 예외를 발생시키는 것이 프론트가 처리하기 편할 지 
2. 빈 리스트를 내려주는 것이 프론트가 처리하기 편할 지
고민해보시면 좋을 것 같습니다.

저는 2번쪽으로 해주는 것이 편하다고 생각하고, 프론트 분들도 편하다고 해서 아래처럼 빈 것인지 체크하지 않고 바로 내려주는 쪽으로 진행했습니다.
참고하시면 될 것 같아요!
```suggestion
    // TODO : 회원 이름으로 회원 정보를 조회한다.
    public List<MemberResponse> searchMembersWithName(String memberName) {
        // DB에서 memberName에 해당하는 회원 정보를 조회한다.
        List<Member> members = memberRepository.findByMemberName(memberName);
        
        // 조회된 회원 정보를 List<MemberResponse>로 변환하여 반환한다.
        return members.stream()
                .map(MemberResponse::from)
                .toList();
    }
```